src/Unleash/Utilities/ToggleBootstrapFileProvider.cs

```diff
@@ -15,7 +16,14 @@ internal ToggleBootstrapFileProvider(string filePath, UnleashSettings settings)
 
         public string Read()
         {
-            return settings.FileSystem.ReadAllText(filePath);
+            try
+            {
+                return settings.FileSystem.ReadAllText(filePath);
+            }
+            catch (FileNotFoundException)
+            {
+                return string.Empty;
+            }
```

Comment:
[Docs](https://learn.microsoft.com/en-us/dotnet/api/system.io.file.readalltext?view=net-9.0) say this raises a FileNotFoundException if this isn't found, which matches my experience of this API. I've chosen to make it work like the test says it should but I honestly have no idea how this ever passed. Pretty sure it shouldn't have